OpenLIFUData/OpenLIFUData.py

```diff
@@ -736,6 +813,30 @@ def clear_plan(self,  clean_up_scene:bool = True) -> None:
         if clean_up_scene:
             plan.clear_nodes()
         self.getParameterNode().loaded_plan = None
+    
+    @display_errors
+    def add_subject_to_database(self, subject_name, subject_id): 
+        """ Adds new subject to loaded openlifu database.
+
+        Args:
+            subject_name: name of subject to be added (str)
+            subject_id: id of subject to be added (str)
+        """
+
+        newOpenLIFUSubject = openlifu_lz().db.subject.Subject()
+        newOpenLIFUSubject.name = subject_name
+        newOpenLIFUSubject.id = subject_id
+
+        subject_ids = self.db.get_subject_ids()
+
+        if newOpenLIFUSubject.id in subject_ids:
+            if not slicer.util.confirmYesNoDisplay(
+                f"Subject with ID {newOpenLIFUSubject.id} already exists in the database. Overwrite subject?",
+                "Subject already exists"
+            ):
+                return
+
+        self.db.write_subject(newOpenLIFUSubject, on_conflict = 'overwrite')
```

Comment:
This doesn't work in the version of openlifu that is indicated in the `python-requirements.txt` file. You will need to update the version in `python-requirements.txt` so that installing the python requirements using the button in the home module makes everything work. (if you go with the latest I think it will become `on_conflict=OnConlfictOpts.OVERWRITE` or something like that... you will see)